pkg/elasticsearch/indices.go

```diff
@@ -95,9 +96,19 @@ func (ec *esClient) AddAliasForOldIndices() bool {
 		}
 
 		if payload.ResponseBody[index] != nil {
-			indexBody := payload.ResponseBody[index].(map[string]interface{})
+			indexBody, ok := payload.ResponseBody[index].(map[string]interface{})
+			if !ok {
+				logger.Warnf("unable to unmarshal index response body. Type: %s",
```

Comment:
Please add more info as whats going on here and to what. something like: 

`logger.Warnf("unable to unmarshal response body of cluster %q index %q to add alias . Type: %s",...."`

same with the other message

Reply:
Fixed